packages/cache-map/test/index.test.js

```diff
@@ -139,6 +139,96 @@ test('makeCacheMapKit (strong)', testMakeCacheMapKit, {
   keys: [{}, Symbol('unique symbol'), 'string key'],
 });
 
+test('makeCacheMapKit(1)', t => {
```

Comment:
Nit: I would love to see a deletion test in the 1 capacity case too.

Reply:
Added.